mne-cpp.pri

```diff
@@ -219,3 +219,7 @@ isEmpty( MNE_INSTALL_INCLUDE_DIR ) {
 # FFTW dir
 FFTW_DIR_LIBS = $$shell_path(C:\fftw-3.3.5-dll64)
 FFTW_DIR_INCLUDE = $$shell_path(C:\fftw-3.3.5-dll64)
+
```

Comment:
Please do not use CONFIG but MNECPP_CONFIG and move to line 132 and add the following:

`## To build MNE Scan to support LSL data streams: qmake MNECPP_CONFIG+=useLSL`